README.md

```diff
@@ -1,117 +1,121 @@
-lookout SDK [![GoDoc](https://godoc.org/gopkg.in/src-d/lookout-sdk.v0?status.svg)](https://godoc.org/gopkg.in/src-d/lookout-sdk.v0/pb) [![PyPI version](https://badge.fury.io/py/lookout-sdk.svg)](https://pypi.org/project/lookout-sdk/) [![Build Status](https://travis-ci.org/src-d/lookout-sdk.svg)](https://travis-ci.org/src-d/lookout-sdk)
------------
+<h1 align="center">
```

Comment:
according to our CTO html in README should be avoided:
https://github.com/src-d/backlog/issues/1318#issuecomment-435921226

Reply:
I also read https://github.com/src-d/guide/pull/292#issuecomment-436597861 
>_Engine has this version because is a product [...]_

So since **source{d} lookout** is also a product, I just coppied the header from **source{d} engine**.

Has @src-d/product any guidelines about product's `README.md`?

Reply:
I think we need input from @campoy or @mcuadros to move forward with it.

Reply:
I prefer the html headers, but @mcuadros was quite clear on rejecting its use.
I'd say the safest route would be to keep `lookout-sdk` without html, since it's a lib, and use html only in lookout.

Reply:
I understood that **source{d} lookout** would be considered as a whole product, regardless it is separated in different repos, apps, libraries...
But, I'm ok with drop the HTML, that is hard to maintain, and change it again if @src-d/product requires it :dagger:

Reply:
I understood that **source{d} lookout** would be considered as a whole product, regardless it is separated in different repos, apps, libraries...
But, I'm ok with drop the HTML, that is hard to maintain, and change it again if @src-d/product requires it :dagger: